### DISCUSSION TITLE
Update Segment Icon with New Logomark

### DISCUSSION THREAD

**Discussion by brannytempy** [twilio-labs/paste#2855] [Request: Icon] [answered]
Hello Paste!

I'm working with Segment to replace all old logos with new versions. Paste has a Segment logo and will need to be part of that update. I've made updates to the existing icon in Figma and will be posting that for product design review. I am filling a few more icon gaps for Segment overall, so I made a unique discussion for this individual one.

[Figma Branch](https://www.figma.com/file/E6KUvMhioUmAgN0nwmReTM/branch/PaxaIfHYTWMzyRaWdmaRBc/Paste-Components?node-id=6692%3A133288&t=bb2kfMPjwqN4KQxu-1)

Here's the new SVG code directly:

`<svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" fill="none" viewBox="0 0 20 20">
  <path fill="#606B85" d="M11.6 4.25C11.1 4.1 10.55 4 10 4 7.55 4 5.5 5.45 4.55 7.55c-.1.25-.35.35-.6.25L3 7.45c-.3-.1-.4-.45-.3-.7C3.95 3.95 6.75 2 10 2c.75 0 1.5.1 2.2.35.3.1.45.4.35.65l-.35.95c-.1.25-.35.35-.6.3Zm3.9.25a1 1 0 1 1-2 0 1 1 0 0 1 2 0ZM9.05 9h7.9c.3 0 .55-.25.6-.55V7.5c0-.3-.25-.55-.55-.55H9.05c-.3 0-.55.25-.55.55v.95c0 .3.25.55.55.55Zm-6 4h7.9c.3 0 .55-.25.6-.5v-.95c0-.3-.25-.55-.55-.55H3.05c-.3 0-.55.25-.55.5v.95c0 .3.25.55.55.55Zm2.45 3.5a1 1 0 1 0 0-2 1 1 0 0 0 0 2ZM10 16c-.55 0-1.1-.1-1.6-.25-.25-.05-.5.05-.6.3l-.35.95c-.1.25.05.55.35.65.7.25 1.45.35 2.2.35 3.25 0 6.05-1.95 7.3-4.75.1-.25 0-.6-.3-.7l-.95-.35c-.25-.1-.5 0-.6.25-.95 2.1-3 3.55-5.45 3.55Z"/>
</svg>`

And also attached here:
![logo_product_Segment](https://user-images.githubusercontent.com/1590723/205349376-8eb7a471-3c00-4e05-9476-e75cd8fce9c9.svg)

**Comment by serifluous** [comment, marked as answer]
Just approved and merged this in Figma. I followed up in Slack, but to round it out here, too:

Let us know if you want us to update this icon in code, or if you'd want to submit a PR to do it yourself!

**Reply by serifluous** [reply]
@gloriliale will be taking this on over the coming week!

**Reply by nkrantz** [reply]
#2894 in case you want to follow along - should be in soon

**Reply by gloriliale** [reply]
Hi @brannytempy! The updated segment icon is now available in the latest version of Paste!
